public/static/js/settings.js

```diff
@@ -80,10 +81,49 @@ mciModule.controller('SettingsCtrl', ['$scope', '$http', '$window', 'notificatio
       });
   }
 
-  $scope.updateUserSettings = function(new_tz, use_spruce_options) {
+  $scope.showAdditionalFields = function(){
+    var additionalFieldCount = 1;
```

Comment:
You shouldn't need to do this yourself - conditionally setting ng-hide on the entire feedback form will do the same thing